src/Analysis/GWASWizard/GWASQuantitative.jsx

```diff
@@ -115,101 +138,84 @@ const GWASQuantitative = ({ resetGWASType, refreshWorkflows }) => {
-      return (
-        <React.Fragment>
-          <h4 className='GWASUI-selectInstruction'>In this step, you may review the metadata selected for the study, give a name to the study, and submit the GWAS for analysis.</h4>
-          <h4 className='GWASUI-selectInstruction'>Upon submission you may review the status of the job in the ‘Submitted Job Status’ in this App above the enumerated steps</h4>
-          <div className='GWASUI-mainArea'>
-            <GWASFormSubmit
+        );
+      case 4:
+        return (
+          <React.Fragment>
+            <WorkflowParameters
+              quantitativeCohortDefinitionId={selectedCohort.cohort_definition_id}
+              selectedCovariates={selectedCovariates}
+              selectedDichotomousCovariates={selectedDichotomousCovariates}
               sourceId={sourceId}
+              workflowType={'quantitative'}
               numOfPC={numOfPC}
+              handleNumOfPC={handleNumOfPC}
               mafThreshold={mafThreshold}
+              handleMaf={handleMaf}
               imputationScore={imputationScore}
+              handleImputation={handleImputation}
               selectedHare={selectedHare}
-              selectedQuantitativeCohort={selectedCohort}
-              workflowType={'quantitative'}
-              outcome={outcome}
-              // selectedCaseCohort={selectedCaseCohort}
-              // selectedControlCohort={selectedControlCohort}
-              selectedCovariates={selectedCovariates}
-              selectedDichotomousCovariates={selectedDichotomousCovariates}
-              gwasName={gwasName}
-              handleGwasNameChange={handleGwasNameChange}
-              resetGWAS={resetQuantitative}
+              handleHareChange={handleHareChange}
+              handleCovariateDelete={handleCovariateDelete}
+              outcomeId={outcome.concept_id}
             />
             <TourButton stepInfo={stepInfo} />
-          </div>
-        </React.Fragment>
-      );
-    default:
-      return <React.Fragment />;
+          </React.Fragment>
+        );
+      case 5:
+        return (
+          <React.Fragment>
+            <h4 className='GWASUI-selectInstruction'>In this step, you may review the metadata selected for the study, give a name to the study, and submit the GWAS for analysis.</h4>
+            <h4 className='GWASUI-selectInstruction'>Upon submission you may review the status of the job in the ‘Submitted Job Status’ in this App above the enumerated steps</h4>
+            <div className='GWASUI-mainArea'>
+              <GWASFormSubmit
+                sourceId={sourceId}
+                numOfPC={numOfPC}
+                mafThreshold={mafThreshold}
+                imputationScore={imputationScore}
+                selectedHare={selectedHare}
+                selectedQuantitativeCohort={selectedCohort}
+                workflowType={'quantitative'}
+                outcome={outcome}
+                // selectedCaseCohort={selectedCaseCohort}
```

Comment:
Does this still need to be commented out @alilahrime ?

Reply:
nope, removed comment